### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/advanced/testing-dependencies.md`

### PR DESCRIPTION
**Pull request by ceb10n**

### CODE REVIEW

File: docs/pt/docs/advanced/testing-dependencies.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+# Testando Dependências com Sobreposição (Overrides)
+
+## Sobreponto dependências durante os testes
```
Comment (wesinalves) [inline docs/pt/docs/advanced/testing-dependencies.md:3]: There is a typo in Sobreponto

**Review by wesinalves — CHANGES_REQUESTED** [review]
Nice job, Just 1 typo found.

**Review by nazarepiedady — APPROVED** [review]

File: docs/pt/docs/advanced/testing-dependencies.md (hunk 1 above)
Comment (ceb10n) [reply docs/pt/docs/advanced/testing-dependencies.md:3]: I'll fix it, thank you ♥️

**Review by cassiobotaro — APPROVED** [review]

**Comment by alejsdev** [comment]
Cool! Thank you for the help @ceb10n @wesinalves @cassiobotaro @nazarepiedady :100: :rocket:

**Pull request merged**